.kamal/kamal_deployment.md

```
@@ -0,0 +1,60 @@
+- Ports 80 and 443 open
+- Domain pointed to the server (for eg, `staging.circuitverse.org`)
+
+## Server Configuration
+Kamal reads the target server from:
+```shell
+SERVER_IP=<your_server_ip>
+```
+
+## Required Secrets
+These secrets must be configured before deploying, you can provide these via Environment variables, Kamal secrets manager
+| Variable                       | Purpose                         |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)